src/analysisd/ar_json.c

```diff
@@ -0,0 +1,80 @@
+/* Copyright (C) 2015-2021, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+*/
+
+#include "config.h"
+#include "format/to_json.h"
+
+#define VERSION "1.0"
```

Comment:
According to https://github.com/wazuh/wazuh/issues/5934#issuecomment-688380312, version has to be an integer.

Reply:
Change accepted.

Reply:
Change accepted.